src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
hmm
I'm sure, you can avoid eslint disables

Reply:
i have to leave two of them because without them I can't make the onClick handler on the label to edit the title